frontend/micro-ui/web/micro-ui-internals/packages/modules/ads/src/components/ADSCancelBooking.js

```diff
@@ -0,0 +1,60 @@
+import React,{useState} from "react";
+import { Modal, Card,CheckBox} from "@nudmcdgnpm/digit-ui-react-components";
+import {useForm } from "react-hook-form";
+
+const Heading = (props) => {
```

Comment:
For this too

Reply:
this file is for cancelling booking from employee side, i have added , more changes will be done by nikhil